applications/RomApplication/tests/thermal_static_test_files/MainKratosROM.py

```diff
@@ -7,22 +7,22 @@
 from KratosMultiphysics.assign_scalar_variable_process import AssignScalarVariableProcess
 from KratosMultiphysics.ConvectionDiffusionApplication.apply_thermal_face_process import ApplyThermalFaceProcess
 from KratosMultiphysics.ConvectionDiffusionApplication.convection_diffusion_analysis import ConvectionDiffusionAnalysis
-from KratosMultiphysics.RomApplication import ConvectionDiffusionROMsolver
+from KratosMultiphysics.RomApplication import convection_diffusion_stationary_rom_solver
 
 import json
 import sys
 
-class ConvDiffROM(ConvectionDiffusionAnalysis):
+class ConvDiffStationaryROM(ConvectionDiffusionAnalysis):

```

Comment:
why are u redefining this in all the mains? i think you have it in the python_scripts, don't you?

Reply:
There is now a RomAnalysis for either StructuralMechanics or ConvectionDiffusion. There the ROM_BASIS is imported to the nodes and the respective python solver is built, by calling the python_solvers_wrapper

Reply:
[https://github.com/KratosMultiphysics/Kratos/blob/ed6d46ab41cee57637c5cf7663a8d409f72e11a4/applications/RomApplication/python_scripts/convection_diffusion_analysis_rom.py](url)